Bottom: cbce487ed2aa589099f0ae1b0517d85066ae5cae
Top:    66b645c3c06e5f84fec7c4b5eb962bfb50fd75f4
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 09:17:44 -0500

Add new dependency, bc, to Dockerfile


---
diff --git a/docker/buildbot-worker-ptxdist/Dockerfile b/docker/buildbot-worker-ptxdist/Dockerfile
index ee6bfeb5..b14d49c4 100644
--- a/docker/buildbot-worker-ptxdist/Dockerfile
+++ b/docker/buildbot-worker-ptxdist/Dockerfile
@@ -22,6 +22,7 @@ RUN apt-get update \
 &&  DEBIAN_FRONTEND=noninteractive \
     apt-get -y install -q \
         autoconf \
+        bc \
         bison \
         build-essential \
         ccache \
